mmv1/third_party/tgc_next/test/assert_test_files.go

```diff
@@ -66,11 +67,12 @@ func BidirectionalConversion(t *testing.T, ignoredFields []string) {
 }
 
 // Tests a single resource
-func testSingleResource(t *testing.T, testName string, testData ResourceTestData, tfDir string, ignoredFields []string, logger *zap.Logger) error {
+func testSingleResource(t *testing.T, testName string, testData ResourceTestData, tfDir string, ignoredFields []string, logger *zap.Logger, primaryResource bool) error {
 	resourceType := testData.ResourceType
+	var tfplan2caiNotSupported, cai2hclNotSupported bool
```

Comment:
There seem to be a lot of double negatives here which I'm finding confusing.

Could we make these variables called `tfplan2caiSupported` and `cai2hclSupported`?

Reply:
Good point. Changed.